packages/discovery-provider/src/models/delisting/track_delist_status.py

```diff
@@ -9,10 +9,10 @@
 class DelistTrackReason(str, enum.Enum):
     DMCA = "DMCA"
     ACR = "ACR"
-    MANUAL = "MANUAL",
-    ACR_COUNTER_NOTICE = "ACR_COUNTER_NOTICE",
-    DMCA_RETRACTION = "DMCA_RETRACTION",
-    DMCA_COUNTER_NOTICE = "DMCA_COUNTER_NOTICE",
+    MANUAL = ("MANUAL",)
+    ACR_COUNTER_NOTICE = ("ACR_COUNTER_NOTICE",)
+    DMCA_RETRACTION = ("DMCA_RETRACTION",)
+    DMCA_COUNTER_NOTICE = ("DMCA_COUNTER_NOTICE",)
```

Comment:
i'm guessing this was linting probably fixing some bad copy/paste from me. can we remove the parentheses and commas?